### DISCUSSION TITLE
kz.amway.com + CloudFlare, Any solutions to hide the bot do not work

### DISCUSSION THREAD

**Discussion by vedmed123** [ultrafunkamsterdam/undetected-chromedriver#1969] [Q&A]
Website kz.amway.com shows a captcha when using undetected-chromedriver as usual:

driver = undetected-chromedriver.Chrome()
driver.get('https://kz.amway.com')

And then blocks access, even if the captcha is passed correctly by user.
All known methods of hiding the bot, do not work. If Chrome is launched from the program by chromedriver - access blocked.
Any ideas, how to solve this problem?

**Comment by devblack** [comment]
Use nodriver

**Reply by vedmed123** [reply]
What is a "nodriver"? --headless mode? In this mode capcha blocks any driver.get() request.

**Reply by mushifali** [reply]
`nodriver` is a new package (official successor of undetected-chromedriver). Running in `--headless` mode could be the reason why it's easily detected as bot.
